src/Frameworks/WebFrameworks/react/src/screens/Users/Users.js

```diff
@@ -1,39 +1,38 @@
-<% if(isStore && (isCrudWithNode||isCrud)){%>import React, {useState,useEffect} from "react";<%}%>
-<% if((isCrudWithNode||isCrud) && isBootstrap) {%>import { Button } from "react-bootstrap";<%}%>
-<% if((isCrudWithNode||isCrud) && isTailWind) {%>import Button from '../../components/atoms/Button';<%}%>
-<% if((isCrudWithNode||isCrud) && isMaterialUI) {%>import { Button,Box } from "@mui/material";<%}%>
-<% if(isStore && (isCrudWithNode||isCrud)){%>import { useSelector, useDispatch } from "react-redux";<%}%>
-<% if(isCrudWithNode||isCrud){%>import { getUsers  <% if(isCrudWithNode) {%> ,deleteUsers<%}%>} from "./users.actions";<%}%>
-<% if(isStore && (isCrudWithNode||isCrud)){%>import { selectAllUsers } from "./users.selectors";<%}%>
-<% if(isCrudWithNode||isCrud) {%>import AddUser from './AddUser';<%}%>
-<% if(isCrudWithNode||isCrud) {%>import DeleteConfirmationModal from './DeleteConfirmationModal';<%}%>
-<% if(isStore && (isCrudWithNode||isCrud)){%> import Table from '../../components/organisms/Table';<%}%>
-<% if(isCrud ||isCrudWithNode) {%>import { actions } from './users.reducer';
+<% if(isStore){%>import React, {useState,useEffect} from "react";<%}%>
+<% if( isBootstrap) {%>import { Button } from "react-bootstrap";<%}%>
+<% if( isTailWind) {%>import Button from '../../components/atoms/Button';<%}%>
+<% if( isMaterialUI) {%>import { Button,Box } from "@mui/material";<%}%>
+<% if(isStore){%>import { useSelector, useDispatch } from "react-redux";<%}%>
+import { getUsers  <% if(isBackEnd) {%> ,deleteUsers<%}%>} from "./users.actions"
+<% if(isStore){%>import { selectAllUsers } from "./users.selectors";<%}%>
+import AddUser from './AddUser'
+import DeleteConfirmationModal from './DeleteConfirmationModal'
+<% if(isStore){%> import Table from '../../components/organisms/Table';<%}%>
```

Comment:
can we keep single If block for "isStore" Check .. and imports statements along with it ?